src/main/java/com/epam/onlineshop/services/UserService.java

```diff
@@ -1,9 +1,11 @@
 package com.epam.onlineshop.services;
 
+import com.epam.onlineshop.entities.Role;
 import com.epam.onlineshop.entities.User;
 
 public interface UserService {
 
-    boolean addNewUser(User user);
-    String signInUser(User user);
+    User addNewUser(User user);
+    boolean existsByUser(User user);
```

Comment:
When we define method which returns boolean it might be better using "is" in the start of method's naming.
Also, you should use object Boolean, not a primitive type

Reply:
@anveslav , Is it necessary for service method ? I say about Boolean instead boolean

Reply:
I'm not sure, lol

Reply:
@goponenkoev, here you can use primitive. Wrapper don't need